src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -324,8 +425,21 @@ class FlaxBertEncoder(nn.Module):
     def setup(self):
         self.layer = FlaxBertLayerCollection(self.config, dtype=self.dtype)
 
-    def __call__(self, hidden_states, attention_mask, deterministic: bool = True):
-        return self.layer(hidden_states, attention_mask, deterministic=deterministic)
+    def __call__(
+        self,
+        hidden_states,
+        attention_mask,
+        deterministic: bool = True,
+        output_attentions: bool = False,
+        output_hidden_states: bool = False,
+    ):
+        return self.layer(
```

Comment:
Shouldn't we have the base output here with the `return_dict` argument?

Reply:
Good point! That's what we do in `modeling_bert.py`, but I don't really agree with it - I think only `PreTrainedModel`' classes should have a `return_dict` input and all `flax.linen.Module` should always return tuples .... Happy to discuss though! What do you think?

Reply:
Also cc @LysandreJik

Reply:
Actually, I don't see big downside to not using `return_dict` => I'll just implement it